.github/workflows/php-tests.yml

```
@@ -38,38 +38,31 @@ jobs:
+      # under the org's runner quota while keeping the support-policy extremes
+      # under test on every PR: PHP 7.4 (runtime floor) and 8.5 (latest, for
+      # upcoming-deprecation signal), WC/WP at L and L-2 (newest and the floor
+      # of the L-2 support policy). 8.2 sits in the PHP middle as the
+      # security-maintenance line. Intermediate WC/WP (L-1) and older PHP
+      # (8.0/8.1/8.3/8.4) are deferred to a planned merge-time full matrix
+      # (PR description, follow-up #1).
       matrix:
-        woocommerce_support_policy: [ 'L', 'L-1', 'L-2' ]
-        wordpress_support_policy:   [ 'L', 'L-1' ]
-        php_support_policy:         [ 'L', 'L-1', 'L-2', 'L-3', '7.4' ]
+        woocommerce_support_policy: [ 'L', 'L-2' ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)